app-modules/authorization/src/Http/Controllers/SocialiteController.php

```diff
@@ -96,9 +96,8 @@ public function callback(SocialiteProvider $provider)
         if ($provider === SocialiteProvider::Azure) {
             try {
                 $request = Http::withToken($socialiteUser->token)
-                    ->retry(3, 500)
                     ->get('https://graph.microsoft.com/v1.0/me/photo/$value')
-                    ->throwIf(fn (Response $response) => $response->failed() && $response->status() !== 404);
+                    ->throw();
```

Comment:
Please revert the changes here, we still want the `retry` as graph endpoints can sometimes be slow to respond and we only want to throw the exception with those checks.